Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/classic/MetaAdInsideScrollViewFragment.kt

```diff
@@ -0,0 +1,72 @@
+package com.taboola.kotlin.examples.screens.classic
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.facebook.ads.NativeAdLayout
+import com.taboola.android.TBLPublisherInfo
+import com.taboola.android.Taboola
+import com.taboola.android.annotations.TBL_PLACEMENT_TYPE
+import com.taboola.android.listeners.TBLClassicListener
+import com.taboola.kotlin.examples.PlacementInfo
+import com.taboola.kotlin.examples.R
+
+class MetaAdInsideScrollViewFragment : Fragment() {
+
+    private var mRootView: View? = null
+    private var adContainerTop: NativeAdLayout? = null
```

Comment:
@voltfmanSlava - Should this be prefixed with m, mAdContainerTop?

Reply:
@tomer-br-taboola no. i need to remove prefix from mRootView. in kotlin we don't use prefix.

Reply:
Done